projects/elife.yaml

```diff
@@ -387,8 +387,8 @@ api-gateway:
                 - "{instance}--cdn-gateway"
             healthcheck:
                 path: /ping-fastly
-                check-interval: 10000
-                timeout: 2000
+                check-interval: 5000
+                timeout: 1000
```

Comment:
https://github.com/elifesciences/builder/pull/310 set this to 2 seconds because not all POPs in the world can reach the US in 1 second

Reply:
Even with shielding on they would still have to reach a us-east POP :thinking:

Reply:
Over faster connections I think. But they still do have the healthcheck to the origin.